unittests/test_utils.hpp

```diff
@@ -110,7 +110,7 @@ auto push_input_trx(appbase::scoped_app& app, sysio::chain::controller& control,
 }
 
 // Push setcode trx to controller and return trx trace
-auto set_code(appbase::scoped_app& app, sysio::chain::controller& control, account_name account, const vector<uint8_t>& wasm) {
+inline auto set_code(appbase::scoped_app& app, sysio::chain::controller& control, account_name account, const vector<uint8_t>& wasm) {
```

Comment:
On `clang-18`, you may be able to drop the inline for non-member methods/functions (not 100%)